i18n/locales/english/trending.json

```diff
@@ -1,62 +1,62 @@
-  "article24title": "Best Way to Learn Python",
-  "article24link": "https://www.freecodecamp.org/news/the-best-way-to-learn-python-python-programming-tutorial-for-beginners/",
-  "article25title": "Word Count in Google Docs",
-  "article25link": "https://www.freecodecamp.org/news/word-count-in-google-docs-tutorial-counting-words-and-characters-in-a-google-doc-or-word-file/",
-  "article26title": "Node Environment Variables",
-  "article26link": "https://www.freecodecamp.org/news/how-to-use-node-environment-variables-with-a-dotenv-file-for-node-js-and-npm/",
-  "article27title": "Event Viewer in Windows 10",
-  "article27link": "https://www.freecodecamp.org/news/event-viewer-how-to-access-the-windows-10-activity-log/",
-  "article28title": "Combine 1st/Last Name Excel",
-  "article28link": "https://www.freecodecamp.org/news/combine-first-last-names-excel/",
-  "article29title": "JavaScript if-else & if-then",
-  "article29link": "https://www.freecodecamp.org/news/javascript-if-else-and-if-then-js-conditional-statements/"
+  "article0title": "CSS Vertical Align",
+  "article0link": "https://www.freecodecamp.org/news/css-vertical-align-how-to-center-a-div-text-or-an-image-example-code/",
+  "article1title": "JavaScript for loop",
+  "article1link": "https://www.freecodecamp.org/news/javascript-for-loop-how-to-loop-through-an-array-in-js/",
+  "article2title": "Google Doodle Games",
+  "article2link": "https://www.freecodecamp.org/news/google-doodle-games-baseball-pacman-and-more/",
+  "article3title": "Excel Text Function",
+  "article3link": "https://www.freecodecamp.org/news/excel-text-function-how-to-convert-a-number-to-text-format/",
+  "article4title": "What is a Hyperlink?",
+  "article4link": "https://www.freecodecamp.org/news/what-is-a-hyperlink-definition-for-html-link-beginners/",
+  "article5title": "SQL Update Statement",
+  "article5link": "https://www.freecodecamp.org/news/sql-update-statement-update-query-in-sql/",
+  "article6title": "CSS Background Image",
+  "article6link": "https://www.freecodecamp.org/news/css-background-image-with-html-example-code/",
+  "article7title": "What is about:blank?",
+  "article7link": "https://www.freecodecamp.org/news/what-is-about-blank-chrome-blank-homepage-guide/",
+  "article8title": "CSS Background Color",
+  "article8link": "https://www.freecodecamp.org/news/css-background-color-how-to-change-the-background-color-in-html/",
+  "article9title": "Basic HTML5 Template",
+  "article9link": "https://www.freecodecamp.org/news/basic-html5-template-boilerplate-code-example/",
+  "article10title": "CSS Border",
+  "article10link": "https://www.freecodecamp.org/news/css-border-style-and-html-code-examples/",
+  "article11title": "SQL Queries",
+  "article11link": "https://www.freecodecamp.org/news/learn-sql-queries-database-query-tutorial-for-beginners/",
+  "article12title": "HTML <a> tag",
```

Comment:
Here's the issue.  The `<a>` is getting parsed as HTML and creating a new element `<a> tag </a>`.  @scissorsneedfoodtoo can we use something like https://www.npmjs.com/package/html-entities to encode these or do we have to do it by hand?

Reply:
Hey @ojeytonwilliams, thanks for flagging me and for pointing that package out.

I'm thinking that we can escape these selectively within the `footer.njk` template itself using the `escape` filter that's built into Nunjucks. 

I originally thought that any HTML tags included in each locale's `*.json` files would automatically be escaped by i18next, but that doesn't seem to be the case here.

I think the easiest thing we can do for these footer titles is to set a variable for each one, then use the built-in Nunjucks `escape` filter to render each title variable to the template:

```
{%- set article12title %}{% t 'trending:article12title' %}{% endset -%}
{{ article12title | escape }}
```

Reply:
Great! Nunjuck's built in escape sounds like the way to go.